components/countdown.tsx

```diff
@@ -8,16 +8,16 @@ const Countdown: FC = () => {
     const minutes = props.minutes > 1 ? 'minutes' : 'minute'
     return (
       <span>
-        {props.days} {day}
+        {/* {props.days} {day + " "} */}
```

Comment:
Advice for the future: when removing things like the countdown, be sure to remove dead code as well.